core/src/main/scala/org/apache/spark/internal/config/Worker.scala

```diff
@@ -30,7 +30,7 @@ private[spark] object Worker {
 
   val WORKER_CLEANUP_ENABLED = ConfigBuilder("spark.worker.cleanup.enabled")
     .booleanConf
-    .createWithDefault(false)
+    .createWithDefault(true)
```

Comment:
why are you changing this default?  Honestly I am much less comfortable merging it with the default changed, as I don't have much experience w/ standalone mode.

Reply:
As ,this commit depend on  WORKER_CLEANUP_ENABLED . While ,we should keep the default value of `spark.worker.cleanup.enabled `= false . But ,We should make it clear in the docs that `spark.worker.cleanup.enabled` should be enabled if spark.shuffle.service.db.enabled is "true”, all right?